dart/dynamics/MeshShape.cpp

```diff
@@ -220,18 +220,15 @@ void MeshShape::setMesh(
   }
 
   mResourceRetriever = _resourceRetriever;
-
-  _updateBoundingBoxDim();
-  updateVolume();
```

Comment:
I believe you consider these are not necessary because `setMesh()` and `setScale()` are both called in the constructor. However, I think we still need these function calls for the case `setMesh()` is called separately and getting volume and bounding box without manually updating them.

Reply:
> I think the redundant computation should be addressed by the lazy-evaluation method.

I didn't see that you started the lazy-evaluation and I started doing it.. But you're faster than me..! :stuck_out_tongue: